packages/graphile-build-pg/src/QueryBuilder.js

```diff
@@ -570,6 +573,16 @@ ${sql.join(
     this.lockEverything();
     return this.compiledData.select.length;
   }
+  buildSelectRaw() {
+    this.lockEverything();
+    return sql.join(
+      this.compiledData.select.map(
+        ([sqlFragment, alias]) =>
+          sql.fragment`${sqlFragment} as ${sql.identifier(alias)}`
+      ),
+      ", "
+    );
+  }
```

Comment:
If we change to `buildNamedChildSelecting` or similar, I think this can be replaced with something much simpler, such as:

```js
buildSelectRaw() {
  return this._fixedSelectionExpression;
}
```

Reply:
What would `this._fixedSelectionExpression` be, and how would it get defined? I'm assuming that it's something different from `this._isSingleFieldSelector`, but I'm not following you on exactly what it _should_ be.